st2actions/st2actions/runners/pythonrunner.py

```diff
@@ -85,13 +88,22 @@ def _set_up_logger(self):
         logger_name = 'actions.python.%s' % (self.__class__.__name__)
         logger = logging.getLogger(logger_name)
 
-        console = stdlib_logging.StreamHandler()
-        console.setLevel(stdlib_logging.DEBUG)
+        formatter = stdlib_logging.Formatter(
+            config.get('log_format', LOG_FORMAT))
 
-        formatter = stdlib_logging.Formatter('%(name)-12s: %(levelname)-8s %(message)s')
+        console = stdlib_logging.StreamHandler()
+        console.setLevel(config.get('log_level', stdlib_logging.DEBUG))
```

Comment:
Wondering if the config file should have separate log levels for console vs. file?